common/lib/dependabot/workspace/git.rb

```diff
@@ -133,7 +133,7 @@ def stash(memo = nil)
 
       sig { params(memo: T.nilable(String)).returns([String, String]) }
       def commit(memo = nil)
-        run_shell_command("git add #{path}")
+        run_shell_command(%(git add "#{path}"), allow_unsafe_shell_command: true)
```

Comment:
This makes me nervous about possibly introducing an injection attack... but I'm not a Ruby expert. 

Is there another way to tell the `git add` command that `path` is a string? 

Something like `run_shell_command("git add \"#{path}\"")` so that the file path is wrapped in quotes when it's passed to `git add`?